addons/hr_recruitment_survey/controllers/survey_user.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.survey.controllers import main
+
+
+class ApplicantSurvey(main.Survey):
```

Comment:
we can call this file main.py like most controllers are called in odoo

Reply:
thanks, i have updated it

Reply:
@sofiagvaladze can u review this PR again